### ISSUE TITLE
Grammar coloring looks incorrect

### ISSUE THREAD

**Issue by Davider-code** [TheArkive/scintilla_ahk2#8]
I am using AHK Version [ 2.0.2] 
system: windows7 64bit
 
open the file scintilla_ Example.ahk syntax coloring appears incorrect, see gif demo:
![a](https://user-images.githubusercontent.com/6350504/231938890-71ca683e-378c-460a-9392-14f59d9a0460.gif)

**Comment by TheArkive** [comment]
Yes there are some issues currently, I'm actually revising this and doing away with the DLL entirely, but it will take time.
